x/gov/queryable.go

```diff
@@ -185,9 +196,9 @@ func queryTally(ctx sdk.Context, path []string, req abci.RequestQuery, keeper Ke
 	// TODO: Dependant on #1914
 
 	var proposalID int64
-	err2 := keeper.cdc.UnmarshalJSON(req.Data, proposalID)
-	if err2 != nil {
-		return []byte{}, sdk.ErrUnknownRequest(fmt.Sprintf("incorrectly formatted request data - %s", err2.Error()))
+	errRes := keeper.cdc.UnmarshalJSON(req.Data, proposalID)
```

Comment:
Please undo all these refactor changes (for now) to use `errRes` and `res` it's bloating this PR - maybe make a refactor PR after this? - Although, I also don't really agree with this modification either - but we can discuss in the next PR